public/locales/fr/translation.json

```diff
@@ -873,7 +873,7 @@
     "dependentLocality": "Localité dépendante",
     "sortingCode": "Code de tri",
     "state": "État/Province",
-    "userRegistrationRequired": "Inscription de l'utilisateur requise",
+    "Is Public": "est public",
```

Comment:
This is not in keeping with the camel case standard. Remove the space, use camel case and update the source code.